crates/oxi-api/src/opts/get_option_value.rs

```diff
@@ -5,7 +5,7 @@ use crate::{Buffer, Window};
 
 /// Options passed to
 /// [`set_option_value()`](crate::set_option_value).
-#[cfg(not(feature = "neovim-nightly"))]
```

Comment:
All these changes can be removed.

I believe the issue is that on Neovim 0.9 (and maybe on 0.8.3 as well but I'm not 100% sure) the correct ordering of the struct fields is `OptionValueOpts { buffer, window, scope }` instead of `OptionValueOpts { buffer, scope, window }`:

```rust
#[cfg(not(feature = "neovim-nightly"))]
#[derive(Clone, Debug, Default)]
#[repr(C)]
pub struct OptionValueOpts {
    buffer: Object,
    window: Object,
    scope: Object,
}
```